### DISCUSSION TITLE
Bold inside the flashcards?

### DISCUSSION THREAD

**Discussion by joao-casali** [st3v3nmw/obsidian-spaced-repetition#879] [Q&A] [answered]
Hello everyone,

I've noticed just today that inside the review cards, the bold content doesn't shows up as bold.

I've some Snippets enabled, but turning all of them off doesn't resolve it.
___
Here's an exemple:
- Inside Obsidian, is normal, just the way I want:
![Inside Obsidian](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/39a2b91c-3780-43e3-9101-faa3cf87f035)

- Inside the Spaced Repetition review window, without any content showed as bold:
![Inside Spaced Repetition](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/7f886467-ba67-427c-8954-4977f728caa8)
___
**Any idea what can be this issue?** 

_FYK: I'm not the most tech savy, but I know the basics._

**Comment by ronzulu** [comment]
Hi @joao-casali 

Best if you also post your actual markdown file and a screenshot of the following settings:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/753f257a-0923-4ec1-be24-469ac113ed6e)

One thing is that for your second list item, the text (presumably "2") has been turned into a "cloze". This means that whatever characters that surround the text, matches the cloze settings.
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/a2e91abc-320f-49ac-89ab-d731bfb09cd3)

Cheers
Ronny

**Comment by joao-casali** [comment]
Hello @ronzulu,

Here's **my settings**:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/7e6106be-c3f9-416e-b07a-036e7adb53d7)
___
Make no mistake, I know that with this settings, any bolded content is transformed into a cloze. I love this feature and want it to remain that way. I just would really like that inside the SAME flashcard, a bolded part, other than the one that is "clozed", would appear as bold.

That helps to memorize multiple parts of information faster, with just one flashcard.
___
Here is a **.md** that contains everything that I sent: [Test.md](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/14320683/Test.md)
___
How it's displayed inside the **regular Obsidian app**:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/7883d457-3d08-4ccd-9a43-151adab2fbb1)
___
Here's how it's displayed **inside the Spaced Repetition**, when using **INLINE** or **MULTILINE**:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/b9e041ed-491d-43c2-86a1-722b68db2b1e)

Unfortunately, all bolded formatting is lost when **reviewing another bold cloze**:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/153332493/386efe31-db4f-4d4d-910f-24aabe26e1fb)

____

- Maybe the way the code is ran automatically "unbolds" all the other bolded parts that arent't the one being currently answered?
- That would explain why the **INLINE** and **MULTILINE** still retain the bolded parts, but not the other way?

Cheers,
João Casali.

**Comment by ronzulu** [comment, marked as answer]
> Maybe the way the code is ran automatically "unbolds" all the other bolded parts that arent't the one being currently answered?

It happens differently, but the outcome is the same.

One idea to achieve what you want would be to:
- enable "Convert curly brackets to clozes" and 
- disable "Convert bolded text to clozes".

You would then need to use something like:
```
Random list:
- Item {{**bolded one**}};
- Item {{**bolded two**}} + *italic*;
- Item {{**bolded three**}} + <u>sub</u>;
```

And this would show up as:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/080e7f9c-19d0-4db2-bc26-bf81bb796687)

Cheers
Ronny

**Comment by joao-casali** [comment]
> One idea to achieve what you want would be to:
enable "Convert curly brackets to clozes" and
disable "Convert bolded text to clozes".

**This works JUST THE WAY I WANT IT. Thanks!!!**

___
I've an inconvenience though, any idea how I could transform more than 2000 flashcards— mostly the bolded type—, in a fast manner? Simply switching the double asterisk ( ** ) for the curly braces isn't straight forward, since there are two symbols for them ( {{ and }} )? 

TY @ronzulu!!!
João Casali

**Reply by joao-casali** [reply]
Well, I ended up changing manually all the flashcards. Took me 4h but managed to do it.
